server/src/main/kotlin/suwayomi/tachidesk/global/impl/GlobalMeta.kt

```diff
@@ -14,7 +12,36 @@ import suwayomi.tachidesk.global.model.table.GlobalMetaTable
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-object GlobalMeta {
+enum class SettingType {
+    META,
+    GLOBAL_UPDATE,
+}
```

Comment:
this file is intended for general meta where settings that would belong to the app itself or shared between anime and manga, move the new stuff to a new API

Reply:
moved